client/a8c-for-agencies/sections/referrals/referrals-list/index.tsx

```diff
@@ -92,7 +124,9 @@ export default function ReferralList( { referrals, dataViewsState, setDataViewsS
 					totalItems: 1,
 					totalPages: 1,
 				},
-				itemFieldId: 'ref',
+				getItemId: ( item: any ) => {
+					return item.id;
+				},
```

Comment:
If I recall well, if the item has the `id` field, that works perfectly with the DataViews component, if not, we have two alternatives:

- `itemFieldId: 'ref'` even it supports nested fields, `itemFieldId: 'site.value.blog_id'`
- Using the `getItemId` callback method.

So in theory, `getItemId` is not required because the item has the `id` field.